mithril-aggregator/src/services/certifier.rs

```diff
@@ -250,7 +255,14 @@ impl CertifierService for MithrilCertifierService {
                 signed_entity_type,
                 protocol_message,
             )
-            .await?;
+            .await
+            .with_context(|| {
+                format!(
+                    "Can not create open message from protocol_message: '{:?}, epoch: '{}''",
```

Comment:
Maybe to get consistent with the other messages of the module:
```
"Certifier can not create open message from protocol_message: '{:?}, epoch: '{}''",
```

Reply:
This has already been fixed.

Reply:
This has already been fixed.